source/administration/config-settings.rst

```diff
@@ -4053,6 +4053,17 @@ Enable Daily Report
 | This feature's ``config.json`` setting is ``"EnableDaily": false`` with options ``true`` and ``false``.                                                              |
 +----------------------------------------------------------------------------------------------------------------------------------------------------------------------+
 
+Batch Size
+^^^^^^^^^^
+
+Set the size of the batches in which posts will be read from the database to generate the compliance report.
+
+This setting is currently not available in the System Console and can only be set in ``config.json``.
+
++-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
```

Comment:
Thanks @maxerenberg! This is a great addition to the docs.

Right now, last line of your addition isn't displaying. This is because it's formatted as table text and a closing pipe is misaligned. View the editable source and turn off line wrapping to see where the fix is needed, based on similar tables. Once this table formatting issue is fixed, this text will display in the docs as intended.

Reply:
Whoops! Should be fixed now.